src/main/java/seedu/address/logic/commands/ChargeCommand.java

```diff
@@ -0,0 +1,104 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collection;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.AttendanceEntry;
+import seedu.address.model.charge.Charge;
+import seedu.address.model.charge.ChargeDate;
+import seedu.address.model.pet.Pet;
+
+
+/**
+ * Computes a month's charge of a pet identified using it's displayed index from the address book.
+ */
+public class ChargeCommand extends Command {
+
+    public static final String COMMAND_WORD = "charge";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Computes a month's charge of the pet identified "
+            + "by the index number used in the last pet listing.\n"
+            + "Parameters: INDEX (must be a positive integer)"
+            + "m/[MM-yyyy] c/[cost]\n"
```

Comment:
```suggestion
            + "m/[MM-yyyy] c/[COST]\n"
```
To be more consistent with our normal syntax.